### DISCUSSION TITLE
When connected to the control hub, how do I set up the camera vision?

### DISCUSSION THREAD

**Discussion by VanTrook** [FIRST-Tech-Challenge/FtcRobotController#406] [Q&A] [answered]
Sorry, I wasn't very clear in the last discussion. I have the USB camera connected to the control hub, but I have tried multiple programs, from other people and my own, and I cannot seem to get the camera to display what it is seeing to the driver hub. I really need to get this to work for my autonomous program I'm working on, does anybody know how I could get this to work?

**Comment by GramGra07** [comment]
One thing you could try is connecting your webcam directly to your computer. One thing I have found to work is using the ftc dashboard. You will have to research this but it might help.

**Comment by gearsincorg** [comment, marked as answer]
1) If your vision code is activated before the WaitForStart() call,  (as in the various built-in Samples) you can use the top right menu on the Driver Hub to select "Stream Preview".  This shows a frame of the live video.  Tap the image to refresh.

2) If you plug an HDMI monitor into the HDMI port on the Control Hub, you can see the live video preview.

**Comment by k754a** [comment]
Ok, first you want to initialize it, then you want to click the three dots in the corner, then select camera stream from the drop-down menu. Make sure the robot is connected and you are on the right script, also make sure the camera is connected and being sensed.
